plugins/webapi/info/plugin.go

```diff
@@ -5,6 +5,7 @@ import (
 	"sort"
 	goSync "sync"
 
+	manaPkg "github.com/iotaledger/goshimmer/packages/mana"
```

Comment:
It is preferable to refer to mana package as `mana` by default.
If we need the plugin, then we can use the less common `manaPlugin`.